ast3/Parser/tokenizer.c

```diff
@@ -1616,7 +1604,7 @@ tok_get(struct tok_state *tok, char **p_start, char **p_end)
         /* async/await parsing block. */
         if (tok->cur - tok->start == 5) {
             /* Current token length is 5. */
-            if (tok->async_def) {
+            if (tok->async_always || tok->async_def) {
```

Comment:
This whole `async/await` logic was pulled out in Python 3.7, right? Probably worth calling that out clearly in the comments.

Reply:
Yes, I had to put it back.